code/modules/antagonists/cult/cult_structures.dm

```diff
@@ -172,7 +172,12 @@
 
 	var/list/choices = list()
 	for(var/item in options)
-		choices[item] = options[item][PREVIEW_IMAGE]
+		var/datum/radial_menu_choice/cultitem = new()
+		cultitem.name = item
+		// cultitem.info = span_cult_italic(options[item][RADIAL_DESC])
```

Comment:
?

Reply:
It's from the other PR I'm just waiting for merges to resolve it